FirebaseMessaging/Sources/FIRMessagingPubSub.m

```diff
@@ -89,17 +88,18 @@ - (void)subscribeWithToken:(NSString *)token
   options = [options fcm_trimNonStringValues];
 
   [self updateSubscriptionWithToken:token
-                                     topic:topic
-                                   options:options
-                              shouldDelete:NO
-                                   handler:^void(NSError *error) {
-                                     handler(error);
-                                   }];
+                              topic:topic
+                            options:options
+                       shouldDelete:NO
+                            handler:^void(NSError *error) {
+                              handler(error);
+                            }];
```

Comment:
nit: Can you pass the handler directly into this parameter instead of wrapping it in another block?

Reply:
Done. Good catch.